src/lemonade/tools/server/llamacpp.py

```
@@ -412,14 +482,15 @@ def event_stream():
         # Non-streaming response
         try:
             # Disable streaming for non-streaming requests
-            response = client.chat.completions.create(**request_dict)
+            response = client.chat.completions.create(**openai_client_params)
 
             # Show telemetry after completion
             telemetry.show_telemetry()
 
             return response
 
         except Exception as e:  # pylint: disable=broad-exception-caught
+            logging.error("Error during chat completion: %s", str(e))
```

This fixes a bug that llamacpp errors were not being reported in the server log